devel/ci/Dockerfile-rawhide

```diff
@@ -1,7 +1,10 @@
-FROM registry.fedoraproject.org/fedora:rawhide
+FROM registry.fedoraproject.org/fedora:35
```

Comment:
This does not seems correct, since rawhide is now f36. We should keep the rawhide label.